lifecycle/src/test/scala/it/pagopa/interop/agreementprocess/lifecycle/AttributesRulesSpec.scala

```diff
@@ -479,62 +476,5 @@ class AttributesRulesSpec extends AnyWordSpecLike {
 
       verifiedAttributesSatisfied(agreement, eService, consumer) shouldBe false
     }
-
-    "return false if at least one single attribute with REVOKE_ON_EXPIRATION is expired" in {
```

Comment:
I think that some tests on expiration should be added.
The remaining tests never check what happens in case of expiration is reached.